keychain/src/methods/register.ts

```diff
@@ -43,7 +43,14 @@ const register =
     controller.store();
     Storage.set(selectors["0.0.3"].active(), address);
 
-    return { address, deviceKey };
+    return { address, deviceKey, keypair };
   };
 
-export default register;
+const saveDeploy = (address: string, hash: string) => {
```

Comment:
we will need to make sure the caller is authorized (i.e. an admin) like:

https://github.com/cartridge-gg/cartridge/blob/main/keychain/src/methods/sessions.ts#L34